lib/PuppeteerSharp.Tests/FrameUtils.cs

```diff
@@ -22,7 +22,7 @@ public static string DumpFrames(PuppeteerSharp.Frame frame, string indentation =
             var result = indentation + Regex.Replace(frame.Url, @":\d{4}", ":<PORT>");
             foreach (var child in frame.ChildFrames)
             {
-                result += Environment.NewLine + DumpFrames(child, "    " + indentation);
+                result += "\n" + DumpFrames(child, "    " + indentation);
```

Comment:
Late to the party, but this makes the tests fail on Windows! Was NewLine not working on Mac or Linux? It should give "\n" on either platforms!

Reply:
The CI is a Windows Machine

Reply:
maybe we should add CI on a linux machine as well?

Reply:
Looks like AppVeyor sets the line feed when checking out files to "\n" which is why it works on the CI.

https://help.appveyor.com/discussions/problems/1119-allow-changing-git-autocrlf-setting

I'll check out a working copy with the proper setting.

Reply:
Yeah. I've just got that in my local VM. We should fix this but in a more pragmatic way. I don't care if the browser returns a \n or a \r\n is not what we need to test.

I hate spending time making test happy.